### PR TITLE
Changed modal on adding admin to city, region from user table

### PR DESCRIPTION
**Pull request by andrii52**
region from user table

dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by sharix4 — APPROVED** [review]

File: src/pages/UserTable/ChangeUserRegionModal.tsx [hunk 1]
```diff
@@ -1,37 +1,77 @@
...
+          regionName={user.regionName}
+          onChange={handleChange}
+        ></AddAdministratorModal>
+        ):
+        (
+          <Modal title="Попередження"
+            visible={showModal}
+            onOk={handleClick}
+            onCancel={handleClick}>
+            <p> Користувач  <b>{user.firstName} {user.lastName} </b>не являється членом округи!{" "}</p>
```
Comment (christykobyletska) [inline src/pages/UserTable/ChangeUserRegionModal.tsx:70]: " не є "*

File: src/pages/UserTable/ChangeUserCityModal.tsx [hunk 2]
```diff
@@ -1,34 +1,76 @@
...
+        cityId={cityId}
+        onChange={handleChange}
+      ></AddAdministratorModal>
+      ):
+      (
+        <Modal title="Попередження"
+          visible={showModal}
+          onOk={handleClick}
+          onCancel={handleClick}>
+          <p> Користувач  <b>{user.firstName} {user.lastName} </b>не являється членом станиці!{" "}</p>
```
Comment (christykobyletska) [inline src/pages/UserTable/ChangeUserCityModal.tsx:69]: не є

File: src/pages/UserTable/ChangeUserRegionModal.tsx (hunk 1 above)
Comment (andrii52) [reply src/pages/UserTable/ChangeUserRegionModal.tsx]: Changed

File: src/pages/UserTable/ChangeUserCityModal.tsx (hunk 2 above)
Comment (andrii52) [reply src/pages/UserTable/ChangeUserCityModal.tsx]: Done

**Review by nicoolle — APPROVED** [review]

**Review by christykobyletska — APPROVED** [review]

**Pull request merged**